extensions/docling/src/manager/connection-manager.ts

```
@@ -0,0 +1,333 @@
+    );
+
+    const chunkFactory: ChunkProviderConnectionFactory = {
+      creationDisplayName: 'Docling Chunk Provider',
+      create: this.factory.bind(this),
+    };
+
+    this.extensionContext.subscriptions.push(this.doclingProvider.setChunkProviderConnectionFactory(chunkFactory));
+  }
+
+  dispose(): void {
+    this.#connections.forEach((entry, _key) => entry.disposable.dispose());
```

Biome (2.4.15)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)